src/_guides/libraries/library-tour.md

```diff
@@ -2369,289 +1646,65 @@ The dart:convert library also has converters for ASCII and ISO-8859-1
 (Latin1). For details, see the [API docs for the dart:convert library.][dart:convert]
 
 
-## dart:mirrors - reflection
-
-The dart:mirrors library provides basic reflection abilities to Dart.
-Use mirrors to query the structure of your program and to dynamically
-invoke functions or methods at runtime.
-
-The dart:mirrors library works in both web apps and command-line apps.
-To use it, import dart:mirrors.
-
-<?code-excerpt "misc/test/library_tour/mirrors_test.dart (import)"?>
```

Comment:
Do you also want to remove `mirrors_test.dart`?

Reply:
If we no longer have a dart:mirrors tour anywhere, then yes.